Jenkinsfile

```diff
@@ -28,6 +28,7 @@ odsPipeline(
   stageScanForSonarqube(context)
   stageStartOpenshiftBuild(context)
   stageDeployToOpenshift(context)
+  stageCollectBuildArtifacts(context)
```

Comment:
Let's remove this from the PR, because once https://github.com/opendevstack/ods-jenkins-shared-library/pull/52 is merged, the latest master of the prov app will automatically have it (and then would do it twice ...)